drivers/google/google.go

```diff
@@ -242,7 +243,11 @@ func (d *Driver) Start() error {
 	if err != nil {
 		return err
 	}
-	return c.createInstance(d)
+	if err = c.createInstance(d); err != nil {
+		return err
+	}
+	d.IPAddress, err = d.GetIP()
```

Comment:
Yep seems reasonable to me since error "bubble up" is the general approach we take.  It should produce output like:

```
Error starting machine: error getting ip: dial TCP timed out
```